apsi_diploma/apsi_diploma/settings.py

```diff
@@ -24,9 +27,9 @@
 SECRET_KEY = "django-insecure-wv8nxxyjd)nvpc8^n#!ahui0nw+$kcbhi6!@bomr=g)t(2uoj6"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["apsi-dyplomowanie.herokuapp.com"]
```

Comment:
Same with this

Reply:
Disagree fully. There is no reason to remove it, since it does not interfere with anything - and if we remove it, then if we don't add it after pushing to master the app won't work.

Reply:
Doesn't this option break opening accessing this app under `localhost`?

Reply:
I tried accessing this and it ran locally, so I guess not